swift/aperture/util.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+private final class StandardErrorOutputStream: TextOutputStream {
+  func write(_ string: String) {
+    FileHandle.standardError.write(string.data(using: .utf8)!)
```

Comment:
@LarsJK Is using `!` unsafe here? It's already guaranteed to be a String because of `_ string: String`. If it's unsafe, how would you handle this?